setup.py

```diff
@@ -18,5 +18,5 @@
 
 
 setuptools.setup(
-    setup_requires=["setuptools>=30.3.0"],
+    setup_requires=["setuptools>=40.1.0"],
```

Comment:
Why?

Reply:
setuptools need 40.1.0 to support native namespace.